byoeb-v1/byoeb/byoeb/scripts/ragas_model_comparison_v2.py

```
@@ -0,0 +1,1048 @@
+        } for r in chunk])
+
+        try:
+            result   = evaluate(dataset=ds, metrics=metrics, run_config=run_cfg)
+            scores_df = result.to_pandas()
+        except Exception as e:
+            print(f"  [{model}] chunk {chunk_idx+1} error: {e}", flush=True)
+            all_scores.extend([{} for _ in chunk])
+            continue
+
+        # Log scores per row immediately after chunk completes
+        for row, row_scores_tuple in zip(chunk, scores_df.itertuples()):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)